integration/e2ecortex/services.go

```diff
@@ -51,14 +51,15 @@ func NewDistributorWithConfigFile(name, consulAddress, configFile string, flags
 		e2e.NewReadinessProbe(httpPort, "/ready", 204),
 		httpPort,
 		grpcPort,
+		nil,
 	)
 }
 
-func NewQuerier(name string, consulAddress string, flags map[string]string, image string) *CortexService {
-	return NewQuerierWithConfigFile(name, consulAddress, "", flags, image)
+func NewQuerier(name string, consulAddress string, flags map[string]string, image string, envVars map[string]string) *CortexService {
```

Comment:
I'm not sure it's worth adding `envVars` to all such functions, considering it's not a commong use case and you could just call `service.SetEnvVars()` in the single integration test where you need it. What's your take?

Reply:
I had started with that approach first but then it made me feel someone could miss setting that env var, while setting all the params in the function is explicitly done. But I think we would mostly continue using dynamo for all the other test so I can use that function for now. We can refactor it again later if we see more needs for setting environment variables.
Thanks!

Reply:
> then it made me feel someone could miss setting that env var

It would miss anyway if that person is not aware of the required env var. If you want to guaraantee it you should take `bigtableAddress` in input, like we do for `consulAddress`, but to me looks a bit over-engineered adding `bigtableAddress` to each of such functions given it's only required by 1 single scenario (contrary to consul).